next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -43,28 +43,24 @@ export function MultiCheckboxFacet({
           const checked = option.selected;
           const value = option.value as FieldValue;
           return (
-            <li key={`${getFilterValueDisplay(option.value)}`}>
-              <div className="flex items-center">
-                <Checkbox
-                  data-transaction-name={`facet - ${label}`}
-                  id={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                  checked={checked}
-                  onClick={() => (checked ? onRemove(value) : onSelect(value))}
-                />
-                <label
-                  className="ml-2 text-sm text-steelgray"
-                  htmlFor={`example_facet_${label}${getFilterValueDisplay(
-                    option.value
-                  )}`}
-                >
-                  {getFilterValueDisplay(option.value)}{" "}
-                  <span className="text-steelgray">
-                    ({option.count.toLocaleString("en")})
-                  </span>
-                </label>
-              </div>
+            <li
+              key={`${getFilterValueDisplay(option.value)}`}
+              className="flex items-center"
+            >
+              <Checkbox
+                data-transaction-name={`facet - ${label}`}
+                aria-label={`${t("filter-by")} ${getFilterValueDisplay(
+                  option.value
+                )} (${option.count.toLocaleString("en")})`}
+                checked={checked}
+                onClick={() => (checked ? onRemove(value) : onSelect(value))}
+              />
+              <span className="ml-2 text-sm text-steelgray" aria-hidden={true}>
+                {getFilterValueDisplay(option.value)}{" "}
+                <span className="text-steelgray">
+                  ({option.count.toLocaleString("en")})
```

Comment:
Rather than using toLocaleString("en") with a hardcoded language, should we just drop it and use `count` directly? Or is there a reason we need it?